samples/cellular/udp/src/main.c

```diff
@@ -105,6 +123,20 @@ static int modem_init(void)
 		return err;
 	}
 
+#if defined(CONFIG_UDP_RAI_ENABLE)
+	/* Enable Access Stratum RAI support.
+	 * Note: The 1.3.x modem firmware release is certified to be compliant with 3GPP Release 13.
+	 * %REL14FEAT enables selected optional features from 3GPP Release 14. The 3GPP Release 14
+	 * features are not GCF or PTCRB conformance certified by Nordic and must be certified
+	 * by MNO before being used in commercial products.
+	 */
+	err = nrf_modem_at_printf("AT%%REL14FEAT=0,1,0,0,0");
```

Comment:
It's a good idea to enable AS-RAI support here, however this will cause an error with the nRF91x1. The reason is that with nRF91x1 Release 14 features are always supported and the `%REL14FEAT` AT command has been removed.

The easiest way to fix this would be to just ignore the return value from `nrf_modem_at_printf()`. Otherwise a compilation or run time check needs to be added to determine if the command should be sent or not. Maybe `CONFIG_SOC_NRF9160` could be used here, although honestly I think those Kconfig options are a bit messed up.

I settled for this kind of check in a couple of places: https://github.com/nrfconnect/sdk-nrf/blob/9f7148ab7efdbbe88cab9d7b48c0191c225a081c/samples/cellular/gnss/src/assistance_minimal.c#L75

Reply:
Thanks!